From: Arturo Borrero Gonzalez <arturo@debian.org>
Subject: Patch to make the suricata build reproducible
 This patch makes some changes to the suricata build to make it reproducible
 .
 Currently, it only filters out the -fdebug-prefix-map CFLAG which embeds
 the build path.
Forwarded: not-needed
Last-Update: 2016-09-05

Index: suricata/configure.ac
===================================================================
--- suricata.orig/configure.ac
+++ suricata/configure.ac
@@ -2523,7 +2523,8 @@ Generic build parameters:
 echo
 echo "$SURICATA_BUILD_CONF"
 echo "printf(" >src/build-info.h
-echo "$SURICATA_BUILD_CONF" | sed -e 's/^/"/' | sed -e 's/$/\\n"/' >>src/build-info.h
+echo "$SURICATA_BUILD_CONF" | sed -e 's/^/"/' | sed -e 's/$/\\n"/' \
+			| sed 's/-fdebug-prefix-map=.*=. //' >>src/build-info.h
 echo ");" >>src/build-info.h
 
 echo "
